gallery/gallery/tool/l10n_cli/README.md

```diff
@@ -0,0 +1,4 @@
+# l10n
```

Comment:
Can you create a ticket to remove the unnecessary double gallery directories, now that there's nothing in the top one.

Reply:
Pierre already did this: https://github.com/material-components/material-components-flutter-gallery/issues/531 :)

Reply:
never mind, I see one was already created https://github.com/material-components/material-components-flutter-gallery/issues/531